shared/message/support/optimisation/NSGA2EvaluatorMessages.proto

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Might be good to rename this `NSGA2Evaluator` since we already know it's a message?